doc/develop/fixing_data.rst

```diff
@@ -1,33 +1,40 @@
 .. _fixing_data:
 
-***********
-Dataset fix
-***********
-
-Some (model) datasets contain (known) errors that would normally prevent them
-from being processed correctly by the ESMValCore. The errors can be in
-the metadata describing the dataset and/or in the actual data.
-Typical examples of such errors are missing or wrong attributes (e.g.
-attribute ''units'' says 1e-9 but data are actually in 1e-6), missing or
-mislabeled coordinates (e.g. ''lev'' instead of ''plev'' or missing
+******************************************
+Adapting to data sources
```

Comment:
I agree that the title could be improved, but I would prefer to keep the words 'fix' and `dataset' in the title, because those are the words that everyone uses when talking about this topic.

Reply:
OK